verde/utils.py

```diff
@@ -201,3 +203,49 @@ def maxabs(*args):
     arrays = [np.atleast_1d(i) for i in args]
     absolute = [np.abs([i.min(), i.max()]).max() for i in arrays]
     return np.max(absolute)
+
+
+def grid_to_table(grid):
+    """
+    Converts an xarray Dataset grid to a Pandas DataFrame
+
```

Comment:
Please add a description of what the function does and what the user should expect from the output. It's good to mention things like: 1. the output coordinates are put through a meshgrid, 2. any restrictions (only works for 2d, or can be ndm, multiple data variables or only a single). 3. possible use cases for this function (to pass along grid data to functions that expect xyz, like `BlockReduce`).

Reply:
Updated pretty much everything. Thank you so much for the constructive feedback. Still having issues getting the documentation build to include the docstring.

Reply:
You'll need to:

1. Import your function on `verde/__init__.py` to make part of the API. This way it can be accessed as `verde.grid_to_table` and we can move it around overcas internally without breaking the API.
2. Include it in `doc/api/index.rst`. This should be enough for Sphinx to pick it up

Reply:
I was missing the import. It looks like it built the documentation locally, I'm going to commit the changes.

Reply:
OK, the docs look good. Now it's a matter of getting the tests passing. I'll leave some comments below.